setup.py

```diff
@@ -61,6 +61,7 @@
     'chardet==3.0.4',
     'astral==1.5',
     'certifi>=2017.4.17',
+    'attrs==17.4.0',
```

Comment:
This introduces attrs to the codebase as per https://github.com/home-assistant/architecture/issues/4